src/pencilkit.cs

```diff
@@ -246,9 +300,171 @@ interface PKToolPicker {
 		[Export ("colorUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle ColorUserInterfaceStyle { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Create individual instances instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Create individual instances instead.")]
 		[Static]
 		[return: NullAllowed]
 		[Export ("sharedToolPickerForWindow:")]
 		PKToolPicker GetSharedToolPicker (UIWindow window);
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("showsDrawingPolicyControls")]
+		bool ShowsDrawingPolicyControls { get; set; }
+
+		// !missing-null-allowed! 'System.Void PencilKit.PKToolPicker::set_AutosaveName(System.String)' is missing an [NullAllowed] on parameter #0
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("stateAutosaveName")]
+		string stateAutosaveName { get; set; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKInk : NSCopying
+	{
+		[Export ("initWithInkType:color:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string type, UIColor color);
+
+		[Export ("inkType")]
+		string InkType { get; }
```

Comment:
Looking at the [documentation](https://developer.apple.com/documentation/pencilkit/pkinkingtool/inktype) InkType is an enum. We should use a better type than a string.

Reply:
Docs on string backed enums https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#enum-attributes